src/lib/DALMeteorMethods/UserMethods.js

```diff
@@ -1,19 +1,104 @@
 /*eslint-disable*/
 Users = new Mongo.Collection.get("users");
 if (Meteor.isServer) {
-    Meteor.publish("users", function(){
-      return Users.find({_id:this.userId});
+    Meteor.publish("users", function() {
+        return Users.find({
+            _id: this.userId
+        });
     });
-}else {
-  Meteor.subscribe("users");
+    Meteor.publish("usersInRoom", function() {
+        return Users.find({});
```

Comment:
We've gone over this before, we can't do this, as it will publish all of the user's info to all of the clients, which includes login tokens and other stuff. THIS IS A BIG NO NO. Also, this file is supposed to be for meteor methods, yet there isn't a single one in here, move most of this stuff into the DALMethods folder, then turn this file into meteor methods, that way, you don't have to publish all the client's info...